rllib/core/models/configs.py

```diff
@@ -216,37 +236,47 @@ class CNNEncoderConfig(ModelConfig):
     Attributes:
```

Comment:
Same comment for using code-blocks in CNN as well.

Reply:
done